Index: index.php
===================================================================
diff --git a/index.php b/index.php
deleted file mode 100644
--- a/index.php	(revision 3ff1d828a241f9781de143e0c0dbcd9511bee6a8)
+++ /dev/null	(revision 3ff1d828a241f9781de143e0c0dbcd9511bee6a8)
@@ -1,1 +0,0 @@
-<?php
Index: public/includes/footer.php
===================================================================
diff --git a/public/includes/footer.php b/public/includes/footer.php
deleted file mode 100644
--- a/public/includes/footer.php	(revision 3ff1d828a241f9781de143e0c0dbcd9511bee6a8)
+++ /dev/null	(revision 3ff1d828a241f9781de143e0c0dbcd9511bee6a8)
@@ -1,1 +0,0 @@
-<?php
Index: public/config/config.php
===================================================================
diff --git a/public/config/config.php b/public/config/config.php
deleted file mode 100644
--- a/public/config/config.php	(revision 3ff1d828a241f9781de143e0c0dbcd9511bee6a8)
+++ /dev/null	(revision 3ff1d828a241f9781de143e0c0dbcd9511bee6a8)
@@ -1,13 +0,0 @@
-<?php
-$DB_HOST = "localhost";
-$DB_USER = "root";
-$DB_PASS = "";
-$DB_NAME = "savouinos";
-$DB_PORT = 3306;
-
-$conn = new mysqli($DB_HOST, $DB_USER, $DB_PASS, $DB_NAME, $DB_PORT);
-
-if ($conn->connect_error) {
-    die("Erreur de connexion MySQL : " . $conn->connect_error);
-}
-?>
Index: public/includes/header.php
===================================================================
diff --git a/public/includes/header.php b/public/includes/header.php
deleted file mode 100644
--- a/public/includes/header.php	(revision 3ff1d828a241f9781de143e0c0dbcd9511bee6a8)
+++ /dev/null	(revision 3ff1d828a241f9781de143e0c0dbcd9511bee6a8)
@@ -1,1 +0,0 @@
-<?php
diff --git a/public/asset/js/js.js b/public/asset/js/js.js
deleted file mode 100644
diff --git a/public/asset/CSS/style.css b/public/asset/CSS/style.css
deleted file mode 100644
